docs/administrator-guide/administrator-slash-commands.md

```diff
@@ -23,6 +23,5 @@ Administrator slash commands are used to perform system-level functions that req
 * `/jira instance unalias [alias-name]` - Remove an alias from an instance
 * `/jira instance list` - List installed Jira instances
 * `/jira instance v2 <jiraURL>` - Set the Jira instance to process \"v2\" webhooks and subscriptions (not prefixed with the instance ID)
-* `/jira stats` - Display usage statistics
```

Comment:
Maybe add the `/jira setup` command, or do we not want to advertise that here?

Reply:
I thought we decided against that, wouldn't be in this PR anyway.